### PR TITLE
Merge pull request #1 from Tirthpatel1568/devlopement

### PR DESCRIPTION
**Pull request by Tirthpatel1568**
REFS: #20 and fixes #20

### CODE REVIEW

**Comment by Tirthpatel1568** [comment]
REFS: #20 and fixes #20-Day 3 complete

File: Modules/Html/Day3/Tirth-Patel/Employee-deatail-form.html [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+        <br>
+        <div>
+            <label for="loc">Location</label>
+            <select name="Location" id="loc" required>
+                <option value="Amd">Ahmedabad</option>
+                <option value="rjk">Bhavnagar</option>
+                <option value="bhav">Rajkot</option>
+                <option value="srt">Surat</option>
+
+
```
Comment (radixdt12020) [inline Modules/Html/Day3/Tirth-Patel/Employee-deatail-form.html:48]: Remove unwanted line break

File: Modules/Html/Day3/Tirth-Patel/Employee-deatail-form.html [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+        </div>
+        <br>
+        <div>
+
+            Age <input type="number" required>
+
+        </div>
+        <br>
+        <div>
+            Gender:
```
Comment (radixdt12020) [inline Modules/Html/Day3/Tirth-Patel/Employee-deatail-form.html:24]: apply name property in the Gender

File: Modules/Html/Day3/Tirth-Patel/Employee-deatail-form.html [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+            Date-Of-Joining: <input type="date" required>
+        </div>
+        <br>
+        <div>
+            Contact number: <input type="tel" pattern="[0-9] {10}" maxlength="10" required>
+
+        </div>
+        <br>
+        <div>
+            <input type="submit">
```
Comment (radixdt12020) [inline Modules/Html/Day3/Tirth-Patel/Employee-deatail-form.html:69]: value for the button is not written

File: Modules/Html/Day3/Tirth-Patel/HRdeparment.html [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>HRdepartment</title>
+    <style>
+        ul li {
+            font-size: x-large;
+        }
+
+        ul li ul li {
```
Comment (radixdt12020) [inline Modules/Html/Day3/Tirth-Patel/HRdeparment.html:13]: why it is twice

File: Modules/Html/Day3/Tirth-Patel/HRdeparment.html [hunk 5]
```diff
@@ -0,0 +1,135 @@
...
+                    </ol>
+
+                </li>
+
+            </ol>
+
+
+
+
+
```
Comment (radixdt12020) [inline Modules/Html/Day3/Tirth-Patel/HRdeparment.html:121]: remove unwanted line breaks

File: Modules/Html/Day3/Tirth-Patel/HRdeparment.html [hunk 6]
```diff
@@ -0,0 +1,135 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>HRdepartment</title>
+    <style>
+        ul li {
```
Comment (radixdt12020) [inline Modules/Html/Day3/Tirth-Patel/HRdeparment.html:9]: work on style ul and li styles. give proper formatting. 
image

File: Modules/Html/Day3/Tirth-Patel/login-Form.html [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+    <title>LOGIN FORM</title>
+    <style>
+       
+    </style>
+</head>
+<body>
+    <h2>login form</h2>
+    <form>
+        <div>
+            Username <input type="text">
```
Comment (radixdt12020) [inline Modules/Html/Day3/Tirth-Patel/login-Form.html:15]: provide id and name property

File: Modules/Html/Day3/Tirth-Patel/student-reportcard.html [hunk 8]
```diff
@@ -0,0 +1,83 @@
...
+
+
+    </table>
+    <tfoot align="center">
+       ***class 10th students detailed scorecard
+    </tfoot>
+
+</body>
+
+</html>
```
Comment (radixdt12020) [inline Modules/Html/Day3/Tirth-Patel/student-reportcard.html:83]: need individual student Report card, not  all the student marksheet. Make it again

**Comment by Tirthpatel1568** [comment]
REFS: #20 and fixes #20-day4 completed(previous had conflicts)

**Comment by Tirthpatel1568** [comment]
REFS: #22 and fixes #22

almost complete layout

**Comment by Tirthpatel1568** [comment]
REFS: #23 and fixes #23-day 6 compled

**Comment by DarshiL7** [comment]
[tirth.docx](https://github.com/rx-training/Training-Jan-2021/files/5903710/tirth.docx)
please work on above tasks.

**Comment by Tirthpatel1568** [comment]
I made the changes and pushed it into my repository kindly check it.
Thank You.

File: Modules/CSS/Day8/Tirth-Patel/layout.html [hunk 9]
```diff
@@ -115,12 +115,12 @@ <h2>Welcome to this site</h2>
...
             <h2>latest Shot</h2>
-            <img src="/pictures/logo7.png" alt="" height="238px" width="100%">
+            <img src="/Day8/Tirth-Patel/pictures/logo7.png " alt="" height="238px" width="100%">
         </div>
         <div class="clients">
             <h2>Our clients</h2>
-           <span> <img src="/pictures/logo9.png" alt="" height="50pxpx" width="22%"><img src="/pictures/logo9.png" alt="" height="50pxpx" width="22%"><img src="/pictures/logo9.png" alt="" height="50pxpx" width="22%"><img src="/pictures/logo9.png" alt="" height="50pxpx" width="22%"></span>
-            <img src="/pictures/logo9.png" height="50px"width=auto>
+           <span> <img src="/Day8/Tirth-Patel/pictures/logo9.png" alt="" height="50pxpx" width="22%"><img src="/Day8/Tirth-Patel/pictures/logo9.png" alt="" height="50pxpx" width="22%"><img src="/Day8/Tirth-Patel/pictures/logo9.png" alt="" height="50pxpx" width="22%"><img src="/Day8/Tirth-Patel/pictures/logo9.png" alt="" height="50pxpx" width="22%"></span>
+            <img src="/Day8/Tirth-Patel/pictures/logo9.png" height="50px"width=auto>
```
Comment (radixdt12020) [inline Modules/CSS/Day8/Tirth-Patel/layout.html:123]: give relative path

**Pull request merged**